src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -47,28 +47,42 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     }
 
     // IFancyZones
-    IFACEMETHODIMP_(void) Run() noexcept;
-    IFACEMETHODIMP_(void) Destroy() noexcept;
+    IFACEMETHODIMP_(void)
+    Run() noexcept;

```

Comment:
Do we really need to use this formatting? I found it less readable.

Reply:
I've tried many clang-format settings to no avail. Looks like clang-format gets confusted with the return-type macros. Since we're not going to need COM interfaces for FancyZones, I can create an issue to remove `IFACEMETHODIMP_` completely.

Reply:
So there's no flag to keep this on the same line?

Reply:
look like it isn't.

Reply:
looks like there isn't.

Reply:
OK.